tests/unit/concrete/batch/batch.tree

```diff
@@ -0,0 +1,24 @@
+Batch_Unit_Concrete_Test
+├── when function does not exist
+│  └── it should revert
+└── when function exists
+   ├── when non state-changing function
```

Comment:
it would be the only place where we have "-" in the test branch (not talking about what _it_ should do)

should we change it to not use "-"?

Reply:
@PaulRBerg made this change so mb he can answer.

Reply:
We can remove the hyphen.